independent-publisher-connectors/URLBae/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3033 @@
+                  "title": "ID"
+                }
+              }
+            },
+            "headers": {
+              "X-RateLimit-Limit": {
+                "description": "X-RateLimit-Limit",
+                "type": "integer"
+              },
+              "X-RateLimit-Remaining": {
+                "description": "X-RateLimit-Remaining",
+                "type": "integer"
+              },
+              "X-RateLimit-Reset": {
+                "description": "X-RateLimit-Reset",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "summary": "Create a pixel",
+        "description": "A pixel can be created using this endpoint. You need to send the pixel type and the tag.",
+        "operationId": "PixelCreate",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "type": {
+                  "type": "string",
+                  "description": "The type.",
+                  "title": "Type"
+                },
+                "name": {
+                  "type": "string",
+                  "description": "The name.",
+                  "title": "Name"
+                },
+                "tag": {
+                  "type": "string",
+                  "description": "The tag.",
+                  "title": "Tag"
+                }
+              },
+              "required": [
+                "name",
+                "tag",
+                "type"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/pixel/{id}/update": {
+      "put": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "error": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "The error code.",
+                  "title": "Error"
+                },
+                "message": {
+                  "type": "string",
+                  "description": "The response message.",
+                  "title": "Message"
+                }
+              }
+            },
+            "headers": {
+              "X-RateLimit-Limit": {
+                "description": "X-RateLimit-Limit",
+                "type": "integer"
+              },
+              "X-RateLimit-Remaining": {
+                "description": "X-RateLimit-Remaining",
+                "type": "integer"
+              },
+              "X-RateLimit-Reset": {
+                "description": "X-RateLimit-Reset",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "summary": "Update a pixel",
+        "description": "To update a your pixels code via the API, you can use this endpoint.",

```

Comment:
All descriptions must be free from grammatical errors.

Reply:
This has been fixed.